pkg/lib/cockpit-po-plugin.js

```diff
@@ -68,8 +68,6 @@ module.exports = class {
         const patterns = [
             // all translations for that page, including manifest.json and *.html
             `pkg/${this.subdir}.*`,
-            // FIXME: https://github.com/cockpit-project/cockpit/issues/13906
-            'pkg/base1/cockpit.js'
```

Comment:
Hmm.. now I actually wonder whether we should keep this hack, and not include the .po separately. That would avoid another browser ←→ ws roundtrip. The timing tests have shown that it makes no measurable difference how big the file is (4 kB or 150 KB took almost the same time)

See https://github.com/cockpit-project/cockpit/pull/16930#discussion_r824575488

Reply:
We resolved/nack'ed the referenced comment, so this comment is obsolete as well.

Reply:
If we stop running ssh and pcp through webpack, then I certainly care less about this, but I still think that this change is correct. Let's have a call?

Reply:
Yes, it's correct (when assuming that we *do* want to load base1 po separately) -- I already closed the conversation